lib/__tests__/findMatchingFiles-test.js

```diff
@@ -64,8 +58,11 @@ import findMatchingFiles from '../findMatchingFiles';
         ];
       });
 
-      it('returns an empty array', () => {
-        expect(subject()).toEqual([]);
+      it('returns an empty array', (done) => {
```

Comment:
I'm not sure if Jest/Jasmine has this feature but in Mocha, you can return a promise in your test for a similar effect, FYI.